.travis.yml

```diff
@@ -16,47 +14,23 @@ env:
     - HOMEBREW_NO_AUTO_UPDATE=1
   matrix:
     - CASK=pe-client-tools
-    - CASK=pe-client-tools-2019.3
-    - CASK=pe-client-tools-2018.1
+    - CASK=pe-client-tools-2019.8
     - CASK=pdk
     - CASK=puppet-agent
-    - CASK=puppet-agent-5
     - CASK=puppet-agent-6
     - CASK=puppet-agent-7
     - CASK=puppet-bolt
     - CASK=puppet-bolt@2
+addons:
+  homebrew:
+    update: true
```

Comment:
a really old version of homebrew was being used when I last touched this. It would certainly be reasonable to try removing it and see if things still work. The issue was a result of the changes when Homebrew went from `brew cask install` to `brew install --cask`

Reply:
meh, looks like brew on 10.14 doesn't support installing with `--cask`, and 11.3 doesn't support installing with `brew cask install`... so it's simpler to keep updating until we remove 10.14 support